### PR TITLE
[HtmlSanitizer] Add docs for the new HtmlSanitizer defaultAction config

### LINKED ISSUE #20004: [HtmlSanitizer] Add support for configuring the default action
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#57399
| PR author(s) | @Seldaek
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by Seldaek**
Fixes #20004

Requires #20019 to be merged for completeness.

### CODE REVIEW

**Comment by Neirda24** [comment]
Hi @Seldaek . Yes I am. did not had the time to finish things yet.

**Comment by Neirda24** [comment]
@Seldaek : here you go : https://github.com/symfony/symfony/pull/57653 was pretty straightforward. I'll take time to add some tests on DI as well because there are none at the moment. Should it be in the same PR or another one to keep things separated ?

**Comment by Seldaek** [comment]
Looks good to me, thanks. I'd say feel free to add tests in the same PR..

File: html_sanitizer.rst [hunk 1]
```diff
@@ -462,6 +462,80 @@ This can also be used to remove elements from the allow list.
...
+                sanitizers:
+                    app.post_sanitizer:
+                        # ...
+
+                        # remove all tags by default, but process their children
+                        default_action: 'block'
+                        # remove <figure> and its children
+                        drop_elements: ['figure']
+
+    .. code-block:: xml
```
Comment (nicolas-grekas) [inline html_sanitizer.rst:427]: https://github.com/symfony/symfony/pull/57653 will target 8.1, XML is gone \o/

Comment (javiereguiluz) [reply html_sanitizer.rst:427]: Yes, we removed it while merging. Thanks.

**Comment by javiereguiluz** [comment]
This was finally merged! Jordi, thanks a lot for this contribution.
